xbmc/pictures/GUIWindowSlideShow.cpp

```diff
@@ -1144,6 +1144,12 @@ void CGUIWindowSlideShow::OnLoadPic(int iPic, int iSlideNumber, const std::strin
       return;
     }
     CLog::Log(LOGDEBUG, "Finished background loading slot %d, %d: %s", iPic, iSlideNumber, m_slides->Get(iSlideNumber)->GetPath().c_str());
+
+    if (CSettings::GetInstance().GetBool(CSettings::SETTING_PICTURES_HIGHQUALITYDOWNSCALING))
```

Comment:
I would have expected this in this in SlideShowPicture where GL specific things a handled. Why should a window be concerened about mipmapping?